package.json

```diff
@@ -1,71 +1,66 @@
-{
-  "name": "@nodesecure/vulnera",
-  "version": "1.8.0",
-  "description": "NodeSecure vulnerabilities strategies",
-  "exports": "./index.js",
-  "scripts": {
-    "lint": "cross-env eslint index.js",
-    "test-only": "cross-env esm-tape-runner 'test/**/*.js' | tap-monkey",
-    "unit-test-only": "cross-env esm-tape-runner 'test/**/*.unit.js' | tap-monkey",
-    "integration-test-only": "cross-env esm-tape-runner 'test/**/*.integration.js' | tap-monkey",
-    "test:typescript": "tsd",
-    "test": "npm run lint && npm run test-only",
-    "test:unit": "npm run lint && npm run unit-test-only",
-    "test:integration": "npm run lint && npm run integration-test-only",
-    "coverage": "c8 -r html npm test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/NodeSecure/vulnera.git"
-  },
-  "keywords": [
-    "npm",
-    "audit",
-    "nodesecure",
-    "vulnerabilities",
-    "vulnerability",
-    "strategies",
-    "strategy",
-    "security",
-    "node",
-    "wg"
-  ],
-  "author": "GENTILHOMME Thomas <gentilhomme.thomas@gmail.com>",
-  "files": [
-    "index.d.ts",
-    "index.js",
-    "types",
-    "src"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/NodeSecure/vulnera/issues"
-  },
-  "homepage": "https://github.com/NodeSecure/vulnera#readme",
-  "devDependencies": {
-    "@nodesecure/eslint-config": "^1.6.0",
-    "@slimio/is": "^2.0.0",
-    "@small-tech/esm-tape-runner": "^2.0.0",
-    "@small-tech/tap-monkey": "^1.4.0",
-    "c8": "^8.0.0",
-    "cross-env": "^7.0.3",
-    "tape": "^5.6.1",
-    "tsd": "^0.29.0"
-  },
-  "type": "module",
-  "dependencies": {
-    "@myunisoft/httpie": "^2.0.1",
-    "@nodesecure/github": "^1.2.0",
-    "@nodesecure/npm-registry-sdk": "^1.4.1",
-    "@npmcli/arborist": "^7.0.0",
-    "@pnpm/audit": "^7.0.12",
-    "@pnpm/lockfile-file": "^8.1.2",
-    "semver": "^7.3.8"
-  },
-  "tsd": {
-    "directory": "test/types",
-    "compilerOptions": {
-      "esModuleInterop": true
-    }
-  }
-}
+{
+  "name": "@nodesecure/vulnera",
+  "version": "1.8.0",
+  "description": "NodeSecure vulnerabilities strategies",
+  "type": "module",
+  "exports": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "lint": "cross-env eslint src/**/*.ts",
+    "test-only": "glob -c \"tsx --test\" \"./test/**/*.spec.ts\"",
+    "unit-test-only": "glob -c \"tsx --test\" \"./test/**/*.unit.spec.ts\"",
+    "integration-test-only": "glob -c \"tsx --test\" \"./test/**/*.integration.spec.ts\"",
+    "test": "npm run lint && npm run test-only",
+    "test:unit": "npm run lint && npm run unit-test-only",
+    "test:integration": "npm run lint && npm run integration-test-only",
+    "coverage": "c8 -r html npm test"
```

Comment:
```suggestion
    "coverage": "c8 -r html npm test",
    "prepublishOnly": "npm run build"
```
It may be useful to add this script to prevent release without compiling TS ?